packages/playback/src/lib/eme/eme-manager.ts

```diff
@@ -66,6 +66,21 @@ export class EmeManager implements IEmeManager {
     // TODO: implement handling of initData
   }
 
+  // public initializeMediaKeys() {
+  //   // Getting this from contrib eme as it seems to fix a bug in older chrome browsers:
+  //   // https://bugs.chromium.org/p/chromium/issues/detail?id=895449
+  //   // It basically fakes the encrypted event to set up media keys.
+  //   // I'm 80 percent we don't want to keep this
+  // }
```

Comment:
Why do we need this here?

Reply:
I _don't think_ we do, but was curious if anyone had more context to why we do it in contrib-eme

Reply:
I am 99% sure this was an implementation detail

Reply:
I think we can remove this